### ISSUE TITLE
Points/plugs chunk loading??

### ISSUE THREAD

**Issue by Dinip12** [Sunekaer/stoneBlock#662]
## Bug Report

Hi, I'm playing stoneblock 1.0.27 and I'm having a "issue" with flux points on my server. This is a semi public server where people by default don't have access to chunkloaders, but chunks with points/plugs are being chunk loaded. There is no option on the config file to disable this. That is a problem for me since I can't have every chunk on every base loaded even for people that are not contributing to the server but are using flux networks to transfer energy and have their base loaded 24/7.

## Steps to Reproduce (for bugs)

My test was a redstone furnace smelting cobble to a drawer and a capacitor bank in 2 different unclaimed and unloaded chunks by ftb utilites and for my surprise after a few time (like 1h30) all the cobble was smelted to stone...

## Client Information

* Modpack Version: 1.0.27
* Server/LAN/Single Player: Server
* Optifine Installed: Nope
* Shaders Enabled: Nope

**Comment by nzminecraftDOTcom** [comment]
Are you sure they load chunks? I don't have that problem on my server, power just stops when the chunk unloads with the flux plug attached to a power source. You may have something else loading chunks etc.

**Comment by Dinip12** [comment]
I've started a singleplayer world, went to aroma cuz that is the place where people place solar panels.
There is a capacitor bank 2 chunks at the left sending the energy and it has been working for the past ~1h. I stayed afk like 20 chunks far from that machine. There is 10x64 and the machine is very slow, it was no speed upgrades, so the speed is like a vanilla furnace.
![image](https://user-images.githubusercontent.com/8941012/49031691-99ac4500-f1a2-11e8-8c25-cd33200a51af.png)
![image](https://user-images.githubusercontent.com/8941012/49031700-a03abc80-f1a2-11e8-9366-0bedc73d8e58.png)
